### PR TITLE
Add video nodes to prometheus.yml

### PR DESCRIPTION
**Pull request by SuperQ**

### CODE REVIEW

**Comment by SuperQ** [comment]
@msuriar this should monitor the video nodes.

File: ansible/files/prometheus/prometheus.yml [hunk 1]
```diff
@@ -21,6 +21,58 @@ scrape_configs:
         - 'tradedefecit.conference.fosdem.net:9100'
         - 'prometheus.conference.fosdem.net:9100'
 
+  - job_name: 'video_node'
```
Comment (brian-brazil) [inline ansible/files/prometheus/prometheus.yml]: Just call these 'node', put them in with the others.

Are you missing a . before your v?

Comment (SuperQ) [inline ansible/files/prometheus/prometheus.yml]: I wanted to group them separately, but I guess it's fine to just put them in the node group.

Comment (SuperQ) [inline ansible/files/prometheus/prometheus.yml]: Ooops, yea, fixed DNS names.

Comment (msuriar) [inline ansible/files/prometheus/prometheus.yml]: Yeah, missing a . before the v - otherwise LGTM

**Comment by SuperQ** [comment]
Fixed issues

**Pull request merged**